content/yaml-quick-start/building-a-unity-app.md

```diff
@@ -185,9 +248,24 @@ You can do this as follows:
 5. In the Indetification section check the 'Override Default Bundle Identifier' option.
 6. Set the Bundle Identifier to match the identifier name you have used in your Apple Developer Program account.
 
+## Configure Android build settings in Unity {#android-build-settings}
 
-## Custom Gradle templates {#custom-gradle-templates}
+Google recommends that Android applications are published to Google Play using the an application bundle (.aab). You should configure the following settings in Unity before building the application bundle:
 
+1. Open Unity and File > Build Settings.
+2. Make sure Android is selected in the Platform section.
+3. Check the Build App Bundle (Google Play) checkbox.
+4. Make sure that Export Project is **not** checked.
+5. Click on the Player Settings button.
+6. Expand 'Other Settings' and check the 'Override Default Package Name' checkbox.
+7. Enter the package name for your app e.g. com.domain.yourappname.
+8. Set the Version number
+9. Put any integer value in the Bundle Version Code. This will be overriden with the build script.
+10. Set the Minimum API Level and Target API Level to **Android 11.0 (API level 30)** which is required for publishing application bundles.
+11. In the Configuration section set 'Scripting Backend' to **IL2CPP**.
+12. In the Target Architectures section check **ARMv7** and **ARM64** to support 64-bit architectures so the app is compliant with the Google Play 64-bit requirement.
+
```

Comment:
There's inconsistency in how you refer to UI elements and sections. Compare: "Check the Build App Bundle (Google Play) checkbox." vs  "Expand 'Other Settings' and check the 'Override Default Package Name' checkbox."
In Codemagic, we usually use bold for UI elements, but since this is not Codemagic UI, any (consistent) approach would be fine.